<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BodyPartEditorBase</class>
 <widget class="QWidget" name="BodyPartEditorBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Bodypart name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>size x:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider0">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd0"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>size y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd1"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>size z:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd2"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>origin x:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd3"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>origin y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd4"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>origin z:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd5"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider0</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcd0</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider1</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcd1</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcd2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider3</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcd3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider4</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcd4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider5</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcd5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
